src/Products/PloneMeeting/tests/testContacts.py

```diff
@@ -475,6 +475,77 @@ def test_pm_MeetingGetItemAbsents(self):
         self.assertEqual(item1.getItemAbsents(), (hp_uid, ))
         self.assertFalse(item2.getItemAbsents())
 
+    def test_pm_print_signatories_by_position(self):
```

Comment:
Please hardcode the expected value in asserts. So we know if somthing breaks the formatting

Reply:
Will do, thanks :+1:

Reply:
Done :smiley: